typescript/mass-replay.md

```diff
@@ -0,0 +1,89 @@
+# Mass Replay Verification API
+
+Users often manually verify that they have not broken compat with existing
+workflow histories when making changes by downloading histories and replaying
+them with a replay worker.
+
+We aim to make that simpler and faster with the below changes, which should
+be similarly applicable to all SDKs but are laid out here in TS first.
+
+For flexibility and compose-ability reasons, it is desirable to separate the
+construction of a set of histories the user might care about from the 
+downloading of those histories and the replay of those histories. Hence
+the base level APIs proposed are (in order):
+
+## Fetching
+See https://github.com/temporalio/proposals/pull/67
+
+Which includes functions for listing workflows according to some query params.
+Those functions return something like `Iterable<WorkflowExecutionInformation>`
```

Comment:
This should be an async iterable as well to support pagination.

Reply:
Heh, yeah I originally had it that way, but the proposal doesn't have an `await` on list.

Reply:
https://github.com/temporalio/proposals/pull/67/files#diff-7d87e4d9af7e84955486e56bc4d577483f5fa34479dbe84b2ceeb50f4d64a81cR66

Reply:
It's a little hard to see with all of the comments.

Reply:
ooooo